api/lib/source/cosmosV0-source.js

```diff
@@ -1,14 +1,15 @@
 const { RESTDataSource, HTTPCache } = require('apollo-datasource-rest')
 const { InMemoryLRUCache } = require('apollo-server-caching')
 const BigNumber = require('bignumber.js')
-const { orderBy, keyBy, uniqBy } = require('lodash')
+const _ = require('lodash')
```

Comment:
does this mean we're importing all of lodash instead of just the functions we need?

Reply:
all of it since we are using 5 different lodash functions. I think it is cleaner like this. Or you think this sacrifices performance too much?

Reply:
on node it doesn't really matter that we import all of lodash.

Reply:
i mean, still seems funny to me - would be nice to reduce dependencies and only import the functions we need - the repo is over 1GB in size

Reply:
but lodash is already there Jordan, in the node_modules in full size